src/GuestPages/Status.js

```diff
@@ -1,17 +1,61 @@
 import React, { useState } from 'react';
 import { Container, Row, InputGroup, FormControl } from 'react-bootstrap';
 import 'bootstrap/dist/css/bootstrap.min.css';
+import axios from 'axios';
 
-const Status = ({ updateStep }) => {
+const { URL } = require('../../config/private.config');
+
+const dummyData = {
+  'guest_four@domain.com': {
+    _id: '61b7b1edda2ddaf0d73c5acc',
+    firstName: 'Guest',
+    lastName: 'Four',
+    email: 'guest_four@domain.com',
+    rsvpStatus: 'attending',
+    group: 'Four Family',
+    event: '61b79b9e0ac02dbe3e12fd1b',
+    __v: 0,
+    rsvpLastUpdated: '2021-12-13T20:50:25.986Z',
+    rsvpNote: "Thanks for the invite! I'll be there.",
+  },
+  'guest_three@domain.com': {
+    _id: '61b7acb5da2ddaf0d73c5aae',
+    firstName: 'Guest',
+    lastName: 'Three',
+    email: 'guest_three@domain.com',
+    rsvpStatus: 'pending',
+    group: 'Three Family',
+    event: '61b79b9e0ac02dbe3e12fd1b',
+    __v: 0,
+  },
+};
+
+const Status = ({ updateStep, updateData }) => {
   const [email, updateEmail] = useState('');
+  const [errorShown, isError] = useState(false);
 
   const checkNextStep = () => {
-    console.log(email);
-    updateStep('new');
-  };
-
-  const toEdit = () => {
-    updateStep('edit');
+    // axios({
```

Comment:
Hey Kyle, it looks pretty good. Consider removing these comments if decided not to use. 
Approved and ready for second review.